### PR TITLE
Handle calculated questions

### PR DESCRIPTION
**Pull request by ngouy**
Handle hidden questions with expressions

### CODE REVIEW

File: src/components/Questionnaire.js [hunk 1]
```diff
@@ -116,23 +135,30 @@ const serializeResults = (results) => {
...
     const serializedQuestionResponses = {
-      id: question.capableId,
+      id: (question || calculatedValue).capableId,
       responses: [],
     };
 
-    if (!Array.isArray(values)) {
-      values = [values];
+    if (calculatedValue) {
+      serializedQuestionResponses["responses"].push({ content: toString(values) });
```
Comment (lvancraen) [inline src/components/Questionnaire.js:150]: hmm where is toString() defined? Am I missing something?

File: src/components/Questionnaire.js [hunk 2]
```diff
@@ -168,9 +194,24 @@ const Questionnaire = ({ survey }) => {
...
+          name: survey_js_question.name,
+          expression: survey_js_question.expression,
+          includeIntoResult: survey_js_question.includeIntoResult,
+          capableId: survey_js_question.capableId,
+          capableType: survey_js_question.capableType,
+        });
+      } else {
+        page_questions.push(survey_js_question);
+      }
+    });
```
Comment (LUMacKenn) [inline src/components/Questionnaire.js:214]: maybe change the snake_case vars here to camelCase?

**Review by LUMacKenn — APPROVED** [review]
Looks good to me, just the one comment

File: src/components/Questionnaire.js (hunk 2 above)
Comment (ngouy) [reply src/components/Questionnaire.js:214]: gotta love the snake_case if you are Rails dev

**Pull request merged**